ledger/accountdb.go

```diff
@@ -1737,6 +1771,26 @@ func (qs *accountsDbQueries) lookupCreator(cidx basics.CreatableIndex, ctype bas
 	return
 }
 
+func (qs *accountsDbQueries) lookupResources(addr basics.Address, aidx basics.CreatableIndex, ctype basics.CreatableType) (data persistedResourcesData, err error) {
+	err = db.Retry(func() error {
+		var buf []byte
+		var rowid sql.NullInt64
+		err := qs.lookupResourcesStmt.QueryRow(aidx, ctype, addr[:]).Scan(&rowid, &data.round, &buf)
+		if err == nil {
```

Comment:
I'm pretty sure this is not what you meant.

Reply:
Yes I had it flipped before, but this is pretty verbatim copied from from lookup() below which has the same structure..